raiden-ts/tests/unit/epics/send.spec.ts

```diff
@@ -115,6 +115,37 @@ describe('send transfer', () => {
     expectChannelsAreInSync([raiden, partner]);
   });
 
+  test('overflow in transfer value is handled', async () => {
+    // expect.assertions(6);
+
+    const max = Two.pow(256) as UInt<32>;
```

Comment:
Nitpick: `2^256` is already outside the `UInt<32>` range. Could you use something smaller (maybe `2^256-1`), but which would fail only after the sum?